common/src/scheduler/mod.rs

```diff
@@ -99,4 +99,6 @@ pub trait SchedulingCostsPerBlock {
     fn waitlist() -> Self::Cost;
     /// Cost for reservation holding.
     fn reservation() -> Self::Cost;
+    /// Cost for storing message in dispatch stash.
```

Comment:
add comment that everything sent delayed goes into dispatch stash

Reply:
done